app/controllers/curation_concerns/permissions_controller.rb

```diff
@@ -0,0 +1,13 @@
+class CurationConcerns::PermissionsController < ApplicationController
+  include CurationConcerns::PermissionsControllerBehavior
+
+  def confirm_access
+  end
```

Comment:
@jcoyne I meant the github review workflow.  How it does not hide the old comments on the PR and show the code forever how it was when the comment was made.